plugin/pkg/client/transport/plugins.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugins
+
+import (
+	_ "k8s.io/kubernetes/plugin/pkg/client/transport/gcp"
```

Comment:
We used to use plugin/ to identify things that wouldn't be referenced by
the core but built into different modules.  In practice, we've never
formally used it and people have halfheartedly applied it.

On Wed, Mar 16, 2016 at 4:58 PM, James DeFelice <notifications@github.com>
wrote:

> In plugin/pkg/client/transport/plugins.go
> <https://github.com/kubernetes/kubernetes/pull/23066#discussion_r56413725>
> :
>
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package plugins
> > +
> > +import (
> > +	_ "k8s.io/kubernetes/plugin/pkg/client/transport/gcp"
>
> OK. last time I discussed that with the k8s team no one could tell me why
> things went into plugin/ vs. elsewhere
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23066/files/759f65de42125a402f759a8080b0100e8cd7cc9e#r56413725>
>